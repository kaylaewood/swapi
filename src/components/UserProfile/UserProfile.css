.user-profile {
  flex-wrap: wrap;
  margin: auto;
  padding-top: 1vw;
  margin-bottom: -6vw;
  text-align: center;
  color: white;
  width: 80%;
}
