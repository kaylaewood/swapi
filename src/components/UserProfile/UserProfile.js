import React from 'react';
import './UserProfile.css';

function UserProfile({ heading }) {
  return (
    <section className='user-profile'>
      <h2>NAME</h2>
      <h3>LEVEL</h3>
      <p>QUO TEQU OTEQ UOTEQU OTEQUOT EQUOTE QUOTEQ UOTEQUOTEQ UOTEQ UOT EQUOT EQUOTEQUOTE QUO TEQU OTEQUOT EQUOTEQUOTEQUOTEQ UOTE QUOTEQUOTE QUOTEQUO TE QUOTEQUOTEQU OTEQUOTE QUOTEQUO  TEQUOTE QUOTEQ UOTE QUOTEQU  OTEQ UOTEQU OTEQ UOTEQUOTE QUOTEQ </p>
    </section>
  );
}

export default UserProfile;
