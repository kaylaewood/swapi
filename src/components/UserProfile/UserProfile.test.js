import React from 'react';
import ReactDOM from 'react-dom';
import UserProfile from './UserProfile';
import { shallow } from 'enzyme';

describe('UserProfile', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<UserProfile />)
    expect(wrapper).toMatchSnapshot();
    })
})
