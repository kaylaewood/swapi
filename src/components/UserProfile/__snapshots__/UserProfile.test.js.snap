// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserProfile should match the snapshot 1`] = `
<section
  className="user-profile"
>
  <h2 />
  <h3 />
  <p />
</section>
`;
